Source/WebCore/crypto/cocoa/CryptoAlgorithmECDSAMac.cpp

```diff
@@ -53,17 +53,18 @@ static PAL::HashFunction toCKHashFunction(CryptoAlgorithmIdentifier hash)
     default:
         ASSERT_NOT_REACHED();
         return PAL::HashFunction::sha512();
+        break;
```

Comment:
break after return is unreachable.  Could you also remove the default above, then put ASSERT_NOT_REACHED(); and return sha12 after the switch for if an invalid value hits the switch?  Removing "default" gives a compiler warning if we've missed a case.